### PR TITLE
Improvement/HOLO 639 check mint event is part of expected environment

### PR DESCRIPTION
**Pull request by BravoNatalie**
## Describe Changes

- changed `indexer/index.ts` 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] I have checked eslint

### CODE REVIEW

File: src/commands/indexer/index.ts [hunk 1]
```diff
@@ -1187,6 +1187,7 @@ export default class Indexer extends HealthCheck {
     )
   }
 
+  // TODO: aqui
```
Comment (alexanderattar) [inline src/commands/indexer/index.ts]: guessing this is a temporary comment?

File: src/commands/indexer/index.ts [hunk 2]
```diff
@@ -1205,6 +1206,14 @@ export default class Indexer extends HealthCheck {
       }`,
       tags,
     )
+
+    const isHolographable = await this.networkMonitor.registryContract.isHolographedContract(contractAddress)
```
Comment (alexanderattar) [inline src/commands/indexer/index.ts:1210]: Cool i think this should do the trick

File: src/commands/indexer/index.ts (hunk 1 above)
Comment (BravoNatalie) [reply src/commands/indexer/index.ts]: oh yeah, I forgot about it

**Review by alexanderattar — APPROVED** [review]

**Review by sogoiii — APPROVED** [review]
👍

**Pull request merged**